### ISSUE TITLE
Make CLI repo hacktoberfest ready.

### ISSUE THREAD

**Issue by Souvikns** [asyncapi/cli#82]
### Why we should take part in Hacktoberfest?
▶ [Check here](https://hacktoberfest.digitalocean.com/resources/companies)

**With Hacktoberfest right around the corner, we need to prepare the repository. Things we need to do** - 

- [ ] Add `Hacktoberfest` topic to the repository
- [ ] Create Suitable labels for hacktoberfest 
  - `Hacktoberfest`
  - `hacktoberfest-accepted` 
  - `spam`
 - [ ] Update the readme to let others know that we are taking part in hacktoberfest and how they can find suitable issues to contribute. 
-  [ ] Open up beginner-friendly issues (suggestion required)

**Comment by derberg** [comment]
@Souvikns yes! the most critical is 1 and 4 bullet point. Labels can be added ad hoc once they are needed. Readme update is not that important, more important is 👇🏼 

have you seen this -> https://groups.google.com/u/1/g/asyncapi-tsc-members/c/SA90QCFB-TI ?

**Comment by Souvikns** [comment]
> have you seen this -> https://groups.google.com/u/1/g/asyncapi-tsc-members/c/SA90QCFB-TI ?

I missed the email thanks for reminding me, will update the sheets when I create issues.
